### PR TITLE
EOS-22846: LR R2 SSPL: Provide DG to CVG mapping Disk Group Alerts

### PR DESCRIPTION
**Pull request by satish-seagate**
Signed-off-by: Satish Darade <satish.darade@seagate.com>

## Coding:

* [ ] Did you follow coding standard and verified it with flake8? 

* [ ] Did your address all Codacy issues?

## Testing:

* [ ] Unittest updated and all unittests are passing?

* [ ] Sanity/Self tests are updated (if applicable)?

* [ ] Manual testing done covering happy path and non-happy path?

## Integration:

* [ ] If there is any interface change, did you communicate it to other components?
* [ ] If there is any interface change, other componenent gate gatekeepers are ready to accept the change?
* [ ] Did you complete deployment test if applicable?

## PR checklist:
* [ ] Did you add Jira number to the PR?

* [ ] DCO and cla-signed?

### CODE REVIEW

File: low-level/sensors/impl/platforms/realstor/realstor_dg_volume_sensor.py [hunk 1]
```diff
@@ -491,6 +496,11 @@ def _create_internal_msg_dg(self, alert_type, disk_group_detail):
                 extended_info.update({key : value})
 
         generic_info.update(extended_info)
+        cvg_info = {
+            "cvg_name": resource_id if resource_id in self.cvg_info_dict else "NA",
+            "cvg_index": self.cvg_info_dict.get(resource_id, "NA")
```
Comment (cdeshmukh) [inline low-level/sensors/impl/platforms/realstor/realstor_dg_volume_sensor.py:501]: as discussed, use cvg_id only as the key

**Review by cdeshmukh — COMMENTED** [review]
capture clear examples of sensor and health data in jira for reference

File: low-level/sensors/impl/platforms/realstor/realstor_dg_volume_sensor.py (hunk 1 above)
Comment (satish-seagate) [reply low-level/sensors/impl/platforms/realstor/realstor_dg_volume_sensor.py]: Done

**Review by cdeshmukh — APPROVED** [review]

**Comment by cdeshmukh** [comment]
> capture clear examples of sensor and health data in jira for reference

[2:43 PM] Satish Darade
    

capture clear examples of sensor and health data in jira for reference

Chetan, I will capture system health data for disk-group through discovery API but for sensor alert data we have to simulate actual alters on hw. We don't have any actuator for disk-group also we don't have sanity for it. So we can only capture disk-group alerts through the actual hw setup.

​[3:07 PM] Chetan Deshmukh
    lets add sensor santiy for disk group then, which will check for cvg name/id too now

**Review by sandeepsgt — APPROVED** [review]

**Review by sumedhak27 — APPROVED** [review]

**Pull request merged**